docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,115 +1,135 @@
-= Contracts for Cairo
+:starknet: https://starkware.co/product/starknet/[Starknet]
+:scarb: https://docs.swmansion.com/scarb[Scarb]
+:installation: https://docs.swmansion.com/scarb/download.html[this guide]
 
-*A library for secure smart contract development* written in Cairo for https://starkware.co/product/starknet/[StarkNet], a decentralized ZK Rollup.
+= Contracts for Cairo
 
-== Usage
+*A library for secure smart contract development* written in Cairo for {starknet}, a decentralized ZK Rollup.
 
 WARNING: This repo contains highly experimental code. Expect rapid iteration. *Use at your own risk.*
 
-=== First time?
+== Overview
+
+=== Installation
 
-Before installing Cairo on your machine, you need to install `gmp`:
+The library is available as a {scarb} package. Follow {installation} for installing Cairo and Scarb on your machine
+before proceeding, and run the following command to check that the installation was successful:
 
 [,bash]
 ----
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-----
+$ scarb --version
 
-TIP: If you have any trouble installing gmp on your Apple M1 computer, https://github.com/OpenZeppelin/nile/issues/22[here's a list of potential solutions].
+scarb 0.7.0 (58cc88efb 2023-08-23)
+cairo: 2.2.0 (https://crates.io/crates/cairo-lang-compiler/2.2.0)
+sierra: 1.3.0
+----
 
-=== Set up your project
+==== Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create an empty directory, then `cd` into it, and initialize a new scarb project by running:
 
 [,bash]
 ----
-mkdir my-project
-cd my-project
-python3 -m venv env
-source env/bin/activate
+mkdir my_project && cd my_project
+scarb init
 ----
 
-Install the https://github.com/OpenZeppelin/nile[Nile] development environment and then run `init` to kickstart a new project.
-Nile will create the project directory structure and install https://www.cairo-lang.org/docs/quickstart.html[the Cairo language], a https://github.com/Shard-Labs/starknet-devnet/[local network], and a https://docs.pytest.org/en/6.2.x/[testing framework].
+The contents of `my_project` should now look like this:
```

Comment:
```suggestion
The contents of `my_project/` should now look like this:
```
we should add `/` at the end of directory names (https://github.com/OpenZeppelin/cairo-contracts/issues/726#issuecomment-1720689263)